README.adoc

```diff
@@ -266,6 +266,43 @@ class UsersController < ApplicationController
 end
 ----
 
+=== Action Filters Order [[action-filters-order]]
+
+Order controller filter declarations in the order in which they will be executed.
+
+[source,ruby]
+----
+# bad
+class UsersController < ApplicationController
+  after_action :after_action_filter
+  around_action :around_action_filter1
+  before_action :before_action_filter1
+  prepend_around_action :prepend_around_action_filter
+  prepend_after_action :prepend_after_action_filter
+  append_around_action :append_around_action_filter
+  append_before_action :append_before_action_filter
+  around_action :around_action_filter2
+  prepend_before_action :prepend_before_action_filter
+  append_after_action :append_after_action_filter
+  before_action :before_action_filter2
+end
+
+# good
+class UsersController < ApplicationController
+  before_action :before_action_filter1
+  before_action :before_action_filter2
+  around_action :around_action_filter1
+  around_action :around_action_filter2
+  after_action :after_action_filter
+  append_before_action :append_before_action_filter
```

Comment:
> Is there a reason to use an append/prepend declarations in the same class as with regular before/after?

Yes, I use them in my project, especially when working with built-in action filters from gems (for example, to run recaptcha checks before devise's built-in action filters). Otherwise, I use action filter without `_append`, `_prepend`.